apps/lobe-chat/1.35.7/docker-compose.yml

```diff
@@ -1,6 +1,6 @@
 services:
   lobe-chat:
-    image: lobehub/lobe-chat:v1.35.7
+    image: lobehub/lobe-chat:v1.35.8
     container_name: ${CONTAINER_NAME}
     restart: always
     networks:
```

Comment:
No significant differences were found, but there is a change from v1.35.7 to v1.35.8 in the version number of "lobehub/lobe-chat". This might be due to the fact that OpenAI released this Docker image on a recent release day or a scheduled update cycle for stability and security reasons. If you need more specific advice about these details or anything else related to Docker containers or versions, please let me know!